data/json/items/gun/762.json

```
@@ -115,5 +115,80 @@
     ],
     "magazines": [ [ "762", [ "762x39_clip" ] ] ],
     "flags": [ "RELOAD_ONE", "NEVER_JAMS" ]
+  },
+  {
+    "id": "wasr10",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "reload_noise_volume": 10,
+    "name": { "str": "WASR-10" },
+    "description": "A semi-automatic-only civilian-legal Kalashnikov-pattern rifle, imported from Romania.  A valuable survivors' commodity.",
+    "weight": "3300 g",
```

```suggestion
    "weight": "3402 g",
```
https://www.centuryarms.com/wasr-144.html
7.5lb = 3402g, can round off if you want.